lang/py/avro/test/test_ipc.py

```diff
@@ -38,6 +40,12 @@ def test_server_with_path(self):
         client_with_default_path = avro.ipc.HTTPTransceiver("apache.org", 80)
         self.assertEqual("/", client_with_default_path.req_resource)
 
+    def test_empty_reader(self):
+        response_reader = avro.ipc.FramedReader(io.BytesIO(b"Bad Response"))
+        with self.assertRaises(avro.errors.ConnectionClosedException) as cm:
+            response_reader.read_framed_message()
+        assert str(cm.exception) == "Reader read 0 bytes."
```

Comment:
I think checking the exact message is unnecessary. Testing for the expected exception type should be sufficient.